### DISCUSSION TITLE
Greetings

### DISCUSSION THREAD

**Discussion by A-M-Ahmed** [dugsiiyeinc/dhaqsoPay#7] [General]
Selayahy @dugsiiyeinc/dhaqsopay-contributors

**Comment by sharafdin** [comment]
> Selayahy @dugsiiyeinc/dhaqsopay-contributors

Waala fiican yahay. dhalinyarada qaar loomaba jeedo shib eeba iska dhaheen

**Comment by Nasrah-muse** [comment]
> Selayahy @dugsiiyeinc/dhaqsopay-contributors

alxamdullilah eng waa la fiicanyahy 🙌
